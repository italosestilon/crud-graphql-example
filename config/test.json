{
    "dbHost": "localhost",
    "dbName": "db",
    "env": "test"
}