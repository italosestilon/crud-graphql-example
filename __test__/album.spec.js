import config from "config";
import Album from "../models/album";

import { connect, clearDatabase, disconnect } from "../test/setup";

import mongoose from "mongoose";

import { sanitizeObject } from "../test/helper";

beforeAll(async () => {
  try {
    await connect();
  } catch (err) {
    console.log(err);
  }
});

beforeEach(async () => {
  try {
    await clearDatabase();
  } catch (err) {
    console.log(err);
  }
});

/*afterAll(async () => {
  try {
    await disconnect();
  } catch (err) {
    console.log(err);
  }
});*/

it("should not save an empty album", async () => {
  try {
    const album = new Album({});
    await album.save();
  } catch (err) {
    console.log(err);
    expect(err).toMatchSnapshot();
  }
});
