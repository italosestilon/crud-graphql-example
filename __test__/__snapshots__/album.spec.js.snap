// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should not save an empty album 1`] = `[TypeError: Cannot read property 'Decimal128' of null]`;
